grpc/dataplane_grpc.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package config;
+
+option go_package = "dataplane/grpc"
+
+service ConfigService {
+    rpc GetGlobalConfig (GetGlobalConfigRequest) returns (GlobalConfig);
+    rpc UpdateGlobalConfig (UpdateGlobalConfigRequest) returns (UpdateGlobalConfigResponse);
+    rpc CheckConfigGeneration (CheckConfigGenerationRequest) returns (CheckConfigGenerationResponse);
+}
+
+message GetGlobalConfigRequest {}
+
+message UpdateGlobalConfigRequest {
+    GlobalConfig config = 1;
+}
+
+message UpdateGlobalConfigResponse {
+    bool status = 1;
+    string message = 2;
+}
+
+message CheckConfigGenerationRequest {}
+
+message CheckConfigGenerationResponse {
+    uint64 generation = 1;
+}
+
+message Subnet {
+    string name = 1;
+    string prefix = 2;
+    uint32 vni = 3;
+}
+
+message VRF {
+    string name = 1;
+    repeated Subnet subnets = 2;
+}
+
+message Interface {
+    string name = 1;
+    uint32 index = 2;
```

Comment:
what does it mean?

Reply:
Interface index is not being set up by the agent. Dataplate operating interfaces using interface index and assigns it on device creation. Agent can obtain it on Get request.